### ISSUE TITLE
I want to finetune the model on a custom dataset. I couldn't find the loss function in the repo. 

### ISSUE THREAD

**Issue by VirajBagal** [facebookresearch/detr#111]
## ❓ How to do something using DETR

Describe what you want to do, including:
1. what inputs you will provide, if any:
2. what outputs you are expecting:

NOTE:

1. Only general answers are provided.
   If you want to ask about "why X did not work", please use the
   [Unexpected behaviors](https://github.com/facebookresearch/detr/issues/new/choose) issue template.

2. About how to implement new models / new dataloader / new training logic, etc., check documentation first.

3. We do not answer general machine learning / computer vision questions that are not specific to DETR, such as how a model works, how to improve your training/make it converge, or what algorithm/methods can be used to achieve X.

**Comment by fmassa** [comment]
For those who stumble on this issue, the Criterion is defined in https://github.com/facebookresearch/detr/blob/10a2c759454930813aeac7af5e779f835dcb75f5/models/detr.py#L83-L97
